src/Commands/User.php

```diff
@@ -53,6 +53,7 @@ class User extends BaseCommand
         shield:user <action> options
 
             shield:user create -n newusername -e newuser@example.com
+            shield:user create -n newusername -e newuser@example.com -g mygroup
```

Comment:
Why not use `-u` instead of `-n` for the username? `-u` is commonly used for username.

Reply:
It seems that the second word is used.
https://github.com/codeigniter4/shield/blob/a0fcea89338c307d282ab163522d7adb93ac0cbf/src/Commands/User.php#L112-L119